### PR TITLE
Suggest TCP/IP tuning to prevent buffer bloat + latency

### PR DESCRIPTION
**Pull request by scanlime**
This seems to be helping lots on my server, where the default
(Debian 9) was to use FIFO queueing which would cause buffer bloat
and huge latency for clients on slower network links.

This patch adds a sysctl.d config file for Linux which sets fq_codel scheduling and BBR congestion control, and adds a section to the production guide on installing this file.

### CODE REVIEW

**Comment by rigelk** [comment]
Nice catch! Could you add a line for ipv6 too?

**Comment by scanlime** [comment]
Good idea! I went looking though, and it seems like this option actually does apply to ipv6 as well as ipv4, it's just badly named. There's no corresponding option or 'tcp' subtree for ipv6, and I found this page offering an explanation with a reference back to the kernel docs,

https://serverfault.com/questions/867056/tcp-congestion-control-for-ipv6-under-linux

**Comment by rigelk** [comment]
Ah, my bad :) Maybe you could add a short note on that in the comments?

File: support/doc/production.md [hunk 1]
```diff
@@ -134,6 +134,19 @@ On FreeBSD you can use [Dehydrated](https://dehydrated.io/) `security/dehydrated
 $ sudo pkg install dehydrated
 ```
 
+### TCP/IP Tuning
```
[the remark this answers is not in the document]
Comment (scanlime) [reply support/doc/production.md]: We could, but it depends on what you mean... I'd call this change optional in that not every distro needs it. But if your distro uses FIFO queueing, this change made a huge difference and streaming was unusable for many people without it.

**Comment by Chocobozzz** [comment]
Thanks @scanlime! :)

File: support/doc/production.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nutomic) [reply support/doc/production.md]: The difference is that Peertube won't even start without the sql extensions. But anyway, it was just a thought.

**Pull request merged**